source/ProSymbolEditor/ProSymbolEditor.csproj

```diff
@@ -11522,6 +11523,11 @@
     </Content>
   </ItemGroup>
   <ItemGroup />
+  <ItemGroup>
+    <Content Include="Files\MilitaryOverlay.lpkx">
```

Comment:
"Files" might be a little too generic (maybe "DeploymentFiles") - but no need to change